src/calculator.js

```diff
@@ -1,47 +1,35 @@
-exports._check = () => {
+exports._check = (num1,num2) => {
+
+  if (typeof num1 !== 'number') {
+    throw new TypeError(`${num1} is not a number`);
+  }
+  if (typeof num2 !== 'number') {
+    throw new TypeError(`${num2} is not a number`);
+  }
+ 
   // DRY up the codebase with this function
   // First, move the duplicate error checking code here
   // Then, invoke this function inside each of the others
   // HINT: you can invoke this function with exports._check()
 };
 
 exports.add = (x, y) => {
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
+  exports._check(x,y)
   return x + y;
 };
 
-exports.subtract = (x, y) => {
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
+exports.subtract = (z, y) => {
+  exports._check(z,y)
   return x - y;
```

Comment:
&#39;x&#39; is not defined.

###### See http://eslint.org/docs/rules/no-undef for details about this rule